#! /bin/sh /usr/share/dpatch/dpatch-run
## -*- diff -*-
## imshow_only_handles_octave_images.dpatch by Thomas Weber <thomas.weber.mail@gmail.com>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: imshow can only handle Octave's native image for the time being
## DP: #478271

@DPATCH@
diff -urNad octave3.0-3.0.1~/scripts/image/imshow.m octave3.0-3.0.1/scripts/image/imshow.m
--- octave3.0-3.0.1~/scripts/image/imshow.m	2008-04-21 18:00:20.000000000 +0200
+++ octave3.0-3.0.1/scripts/image/imshow.m	2008-05-21 13:49:19.000000000 +0200
@@ -37,7 +37,7 @@
 ## image using the supplied color map.
 ##
 ## If a file name is given instead of an image, the file will be read and
-## shown.
+## shown. The file must be in Octave's native image format.
 ##
 ## If given, the parameter @var{string_param1} has value
 ## @var{value1}. @var{string_param1} can be any of the following:
